experimenter/tests/integration/nimbus/android/conftest.py

```diff
@@ -0,0 +1,154 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import os
+import subprocess
+import time
+from pathlib import Path
+
+import pytest
+import requests
+
+from .gradlewbuild import GradlewBuild
+from .models.models import TelemetryModel
+
+here = Path().cwd()
```

Comment:
This gets the current working directory. Do you want the cwd or the directory of the file? (if the latter, `Path(__file__).parent()`)

Also, `cwd` is a classmethod, so you can do `Path.cwd()`.

Reply:
This is so nice, I did not know how to do this. Thanks for the tip!